tests/unit/components/BookingForm.spec.js

```diff
@@ -22,6 +22,22 @@ Vue.component('BadMessage', BadMessage)
 
 const localVue = createLocalVue();
 
+function checkCalledEndpoint(url){
```

Comment:
Slight naming convention:
When calling something "check", I expect the function to return a boolean. Essentially returning has "called endpoint" being checked (which doesn't make much sense).

A better name might be `getCalledEndpoint`